massa-async-pool/src/pool.rs

```diff
@@ -51,6 +51,14 @@ impl AsyncPool {
         }
     }
 
+    /// Resets the pool to its initial state
+    ///
+    /// USED ONLY FOR BOOTSTRAP
```

Comment:
instead of doing this and risking forgetting to do part of it, it might be better to do the following whenever a bootstrap fails:
* completely drop all the structures, including the async pool
* recreate them all from scratch as if the program was just launching

Reply:
@bilboquet is this treated in https://github.com/massalabs/massa/pull/3698 ?

Reply:
ongoing

Reply:
done